examples/basic/poptext/poptext.py

```diff
@@ -0,0 +1,47 @@
+import movis as mv
+import json
+
+scene = mv.layer.Composition(size=(1080, 1900), duration=4.0)
```

Comment:
To align with the consistency of other samples, can you modify the entire line like this?

```python
def main():
    scene = mv.layer.Composition(size=(1080, 1900), duration=4.0)
    ...
    scene.write_video("output.mp4")
```